src/utils.ts

```diff
@@ -18,7 +18,7 @@ export function logDebug(message: string, args: Record<string, any>): void {
  * @param id GTM ID
  * @param params query params object
  */
-export function loadScript(id: string, config: Pick<VueGtmUseOptions, "defer" | "queryParams"> = {}): void {
```

Comment:
`Pick` limited the technically available properties in this method
Please revert to use Pick and just add ` | "compability"` to it